### DISCUSSION TITLE
I try to get the CSS content of a specific global style file by ignoring css-loader and sass-loader.

### DISCUSSION THREAD

**Discussion by mena234** [vercel/next.js#38100] [Help]
### Verify canary release

- [ ] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

- "next": "12.1.0",
- "react": "17.0.2",
- "react-dom": "17.0.2",

### What browser are you using? (if relevant)

Chrome

### Describe the Bug

In the _app.ts file, I try to get the style content of the style.scss file by importing it with ignoring the css-loader and sass-loader, So I can conditionally use it, by creating `style` tag and inserting the content inside it, and add or remove the `style` element on the a specific condition.

`import { default as DashboardTheme } from '!css-loader!sass-loader!public/dashboard/css/style.scss';`

But I had an error "Can't resolve css-loader"

### Expected Behavior

Not getting an error, and DashboardTheme should contain the style of the file.

For example if the style.scss file contains

`* {
  padding: 0;
  margin: 0
}`

So, DashboardTheme  will equal the value of the file content.

`"* {
  padding: 0;
  margin: 0
}"`

### Link to reproduction

not-exist.

### To Reproduce

not-exist.

**Comment by Rocketjm** [comment]
👋
